documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml

```diff
@@ -14,7 +14,5 @@ spec:
         expose:
           type: ClusterIP
       locations:
-        - name: SiteA
-          url: infinispan+xsite://example-clustera-site
         - name: SiteB
-          url: infinispan+xsite://example-clusterb-site
+          url: infinispan+xsite://example-clusterb-site.<namespace>.svc:7900
```

Comment:
7900 is default value and can be omitted

Reply:
@Crumby Do you think we need the default port here?

Reply:
Theoretically it should work without the port but I haven't tried it

Reply:
@dmvolod even if it's slightly unnecessary do you have any objection to including the default value? If @Crumby has tested it with the port then I'd like to include it.

Reply:
Yes, sure, @oraNod let's keep it as is